test/extended/setup.sh

```diff
@@ -2,6 +2,8 @@
 #
 # This abstracts starting up an extended server.
 
+source "${OS_ROOT}/hack/util.sh"
```

Comment:
Hmm... we do seem to be doing that in `conformance.sh` ... could you try debugging when `cleanup_openshift` gets taken out of scope? Or I can look into it if this is blocking all conformance tests right now.